qst/src/main/java/io/deephaven/qst/table/NaturalJoinTable.java

```diff
@@ -15,6 +16,8 @@
 @NodeStyle
 public abstract class NaturalJoinTable extends JoinBase {
 
+    public abstract NaturalJoinType joinType();
```

Comment:
We should likely declare a default here.

Should we deprecate `ExactJoinTable`?

Reply:
Default added, re: deprecation, @cpwright do you have an opinion?

Reply:
I don't imagine we're going to take either the table method or this one away any time soon; so I'm not sure if it is useful or not to deprecate it.